README.md

```diff
@@ -228,7 +228,7 @@ Alternatively, install Starship using any of the following package managers:
 | CentOS 7+          | [Copr]                  | `dnf copr enable atim/starship` <br /> `dnf install starship` |
 | Gentoo             | [Gentoo Packages]       | `emerge app-shells/starship`                                  |
 | Manjaro            |                         | `pacman -S starship`                                          |
-| NixOS              | [nixpkgs]               | `nix-env -iA nixpkgs.starship`                                |
+| NixOS              | [nixpkgs]               | `nix-env -iA nixos.starship`                                  |
```

Comment:
Please also add an "any" entry for nixpkgs that still uses `nixpkgs.starship` since that is more common on non-NixOS nix installations.

Reply:
I think, the best solution is to add both commands, one for Non NixOS and NixOS user.

Reply:
Ok, are you going to add this change to this PR?